### PR TITLE
Patch core docs

### PR DESCRIPTION
**Pull request by GroovinChip**
This PR brings updates to documentation

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Review by sachaarbonel — CHANGES_REQUESTED** [review]
Left a couple of reviews but other than that LGTM

File: packages/stream_feed/lib/src/client/stream_user.dart [hunk 1]
```diff
@@ -21,13 +21,13 @@ class StreamUser with EquatableMixin {
   })  : _secret = secret,
         _userToken = userToken;
 
-  ///User id
+  /// User id
   final String id;
 
-  ///User JWT token
+  /// User JWT
```
[the remark this answers is not in the document]
Comment (GroovinChip) [reply packages/stream_feed/lib/src/client/stream_user.dart:27]: It's just 2 words though?

[the remark this answers is not in the document]
Comment (sachaarbonel) [reply packages/stream_feed/lib/src/client/stream_user.dart:27]: I believe there are other places in the SDK where we have a Token field

**Review by sachaarbonel — APPROVED** [review]
LGTM

**Pull request merged**